components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -284,7 +289,8 @@ - (BOOL)isMobileLandscape {
 }
 
 - (BOOL)shouldPresentFullScreen {
-  return [self isAccessibilityMode] || [self isMobileLandscape];
+  // TODO: Github issue #5828
+  return [self isAccessibilityMode] || [self isMobileLandscape] || _initialDrawerFactor >= 1;
```

Comment:
do we need the _initialDrawerFactor >=1 in this case?

Reply:
Removed